crates/controller/src/main.rs

```diff
@@ -91,7 +91,7 @@ fn get_application_state(pod: &Pod) -> Option<ContainerState> {
 
             if let Some(terminated) = state.terminated.as_ref() {
                 if terminated.reason.as_deref() == Some("Error") {
-                    return Some(ContainerState::Failed)
+                    return Some(ContainerState::Failed);
```

Comment:
Just curious: does this produce a change in behavior or are semicolons optional on `return` expressions?

Reply:
They're optional, but the `cargo clippy` and `cargo fmt` prefer the semicolon in this case.